### PR TITLE
nuck figgers

### PR DESCRIPTION
**Pull request by CallumlousTV**
### [WLM] NIGGA GENERATOR V8.2 CRACKED BY THE SEWER PIG未检测到 HACK UNDETECTED EXPLOIT (BYPASS ANTIVIRUS ANTICHEAT) 写汉字 - 学中文 100% CRACKED REAL NO DOLLAR FREE MONEY NIGGA GENERATOR p你到底是誰. WTF REAL 100% WORKING NO COST FREE GENERATOR (SEWER PIG VERSION 8.2 CRACKED LEAKED APP) (FREE) (BASED) 英雄联盟 NIGGA GENERATOR 20% MONEY BOOSTER SUPER TURBO HACK?? REAL NO DOLLAR DOWNLOAD 一样。卡萨丁尼德勒很 DOWNLOAD 2020 FREE CLEAN 100%пенис обезьяныNIGGA GENERATOR CRACK 2.3 4.1 8.1 8.2 LEAKED 放屁猴子猴子阴茎下水道猪 SEWER PIG CRACKED 8.2 DOWNLOAD SAFE CLEAN NO MALWARE猴子阴茎 2020 2019 2018 2017 100% DOWNLOAD NOW 放屁的猴子(CRACKED PROGRAM SOFTWARE 8.2) LEAKED 大胖子(FREE DOWNLOAD FOR 0 DOLLAR(EVADE ANTIVIRUS AND ANTICHEAT 2020 UNDETECTED)12英寸公鸡一样。卡萨丁尼德勒很 2020 WORKING OFFICIAL $ REAL NO MONEY COST https://bit.ly/3atQ3GV AUTOCOLLECT AUTOMERGE FAST START HACK 瓜雷茲 (2019 VERSION CRACKED) 太小機器人？ DX？ X？回覆 // NO ADWARE REAL (NO RAT) NO-TRACKING (DNC SECRET HACK WORKING 2012)英雄联盟[ FREE GENERATOR BOOST NO VIRUS (UNDETECTED) (OFFICIAL)你到底是誰FAST START HACK 0 DOLLARكرات قرد الدهون الكبيرةNO RAT NO VIRUS FOR NO MONEY DOWNLOAD NO AD大胖猴子球FIX COMPUTER FAST COMPUTER OPTIMISE(FREE DOWNLOAD NO RAT)猴子球FULLY UNDECTED完全未检测到的版本8.2裂纹破解的发电机组未检测到的通过旁路抗病毒抗体NO ANTIVIRUS FIND DOWNLOAD FOR FREE

### CODE REVIEW

File: TODO.md [hunk 1]
```diff
@@ -24,7 +37,68 @@
...
...CKED BY THE SEWER PIG未检测到 HACK UNDETECTED EXPLOIT (BYPASS ANTIVIRUS ANTICHEAT) 写汉字 - 学中文 100% CRACKED REAL NO DOLLAR FREE MONEY NIGGA GENERATOR p你到底是誰. WTF REAL 100% WORKING NO COST FREE GENERATOR (SEWER PIG VERSION 8.2 CRACKED LEAKED APP) (FREE) (BASED) 英雄联盟 NIGGA GENERATOR 20% MONEY BOOSTER SUPER TURBO HACK?? REAL NO DOLLAR DOWNLOAD 一样。卡萨丁尼德勒很 DOWNLOAD 2020 FREE CLEAN 100%пенис обезьяныNIGGA GENERATOR CRACK 2.3 4.1 8.1 8.2 LEAKED 放屁猴子猴子阴茎下水道猪 SEWER PIG CRACKED 8.2 DOWNLOAD SAFE CLEAN NO MALWARE猴子阴茎 2020 2019 2018 2017 100% DOWNLOAD NOW 放屁的猴子(CRACKED PROGRAM SOFTWARE 8.2) LEAKED 大胖子(FREE DOWNLOAD FOR 0 DOLLAR(EVADE ANTIVIRUS AND ANTICHEAT 2020 UNDETECTED)12英寸公鸡一样。卡萨丁尼德勒很 2020 WORKING OFFICIAL $ REAL NO MONEY COST https://bit.ly/3atQ3GV AUTOCOLLECT AUTOMERGE FAST START HACK 瓜雷茲 (2019 VERSION CRACKED) 太小機器人？ DX？ X？回覆 // NO ADWARE REAL (NO RAT) NO-TRACKING (DNC SECRET HACK WORKING 2012)英雄联盟[ FREE GENERATOR BOOST NO VIRUS (UNDETECTED) (OFFICIAL)你到底是誰FAST START HACK 0 DOLLARكرات قرد الدهون الكبيرةNO RAT NO VIRUS FOR NO MONEY DOWNLOAD NO AD大胖猴子球FIX COMPUTER FAST COMPUTER OPTIMISE(FREE DOWNLOAD NO RAT)猴子球FULLY UNDECTED完全未检测到的版本8.2裂纹破解的发电机组未检测到的通过旁路抗病毒抗体NO ANTIVIRUS FIND DOWNLOAD FOR FREE
```
Comment (MaxTCodes) [inline TODO.md:104]: does tis work?

Comment (MaxTCodes) [reply TODO.md:104]: ths*

Comment (MaxTCodes) [reply TODO.md:104]: tish*

Comment (MaxTCodes) [reply TODO.md:104]: tshi

Comment (MaxTCodes) [reply TODO.md:104]: tishd*

Comment (MaxTCodes) [reply TODO.md:104]: shit*

Comment (MaxTCodes) [reply TODO.md:104]: thsiyt*

Comment (MaxTCodes) [reply TODO.md:104]: tishy*

File: TODO.md [hunk 2]
```diff
@@ -3,7 +3,20 @@
 - [ ] Remind Dort to fix bug from 1
 - [ ] Fix auth
 - [x] Double check Gowixx's code
+- [ ] Double check that Eviate's PeniS.
```
[the remark this answers is not in the document]
Comment (CallumlousTV) [reply TODO.md:6]: > does this work?
### 
no

File: TODO.md (hunk 1 above)
Comment (MaxTCodes) [reply TODO.md:104]: t*

Comment (MaxTCodes) [reply TODO.md:104]: h*

Comment (MaxTCodes) [reply TODO.md:104]: i*

Comment (MaxTCodes) [reply TODO.md:104]: s*

Comment (MaxTCodes) [reply TODO.md:104]: THUIS*

Comment (CallumlousTV) [reply TODO.md:104]:
```
NNNNNNNN        NNNNNNNN     OOOOOOOOO     
N:::::::N       N::::::N   OO:::::::::OO   
N::::::::N      N::::::N OO:::::::::::::OO 
N:::::::::N     N::::::NO:::::::OOO:::::::O
N::::::::::N    N::::::NO::::::O   O::::::O
N:::::::::::N   N::::::NO:::::O     O:::::O
N:::::::N::::N  N::::::NO:::::O     O:::::O
N::::::N N::::N N::::::NO:::::O     O:::::O
N::::::N  N::::N:::::::NO:::::O     O:::::O
N::::::N   N:::::::::::NO:::::O     O:::::O
N::::::N    N::::::::::NO:::::O     O:::::O
N::::::N     N:::::::::NO::::::O   O::::::O
N::::::N      N::::::::NO:::::::OOO:::::::O
N::::::N       N:::::::N OO:::::::::::::OO 
N::::::N        N::::::N   OO:::::::::OO   
NNNNNNNN         NNNNNNN     OOOOOOOOO     
                                           
                                           
                                           
                                           
                                           
                                           
                                           
```

Comment (MaxTCodes) [reply TODO.md:104]: > ```
> NNNNNNNN        NNNNNNNN     OOOOOOOOO     
> N:::::::N       N::::::N   OO:::::::::OO   
> N::::::::N      N::::::N OO:::::::::::::OO 
> N:::::::::N     N::::::NO:::::::OOO:::::::O
> N::::::::::N    N::::::NO::::::O   O::::::O
> N:::::::::::N   N::::::NO:::::O     O:::::O
> N:::::::N::::N  N::::::NO:::::O     O:::::O
> N::::::N N::::N N::::::NO:::::O     O:::::O
> N::::::N  N::::N:::::::NO:::::O     O:::::O
> N::::::N   N:::::::::::NO:::::O     O:::::O
> N::::::N    N::::::::::NO:::::O     O:::::O
> N::::::N     N:::::::::NO::::::O   O::::::O
> N::::::N      N::::::::NO:::::::OOO:::::::O
> N::::::N       N:::::::N OO:::::::::::::OO 
> N::::::N        N::::::N   OO:::::::::OO   
> NNNNNNNN         NNNNNNN     OOOOOOOOO     
>                                            
>                                            
>                                            
>                                            
>                                            
>                                            
>                                            
> ```

this*

**Comment by PhoneTheGreat** [comment]
I DID NOT ASK MAX.

**Comment by CallumlousTV** [comment]
> I DID NOT ASK MAX.
 ________  _______    ______   __        __        ________  _______  
/        |/       \  /      \ /  |      /  |      /        |/       \ 
$$$$$$$$/ $$$$$$$  |/$$$$$$  |$$ |      $$ |      $$$$$$$$/ $$$$$$$  |
   $$ |   $$ |__$$ |$$ |  $$ |$$ |      $$ |      $$ |__    $$ |  $$ |
   $$ |   $$    $$< $$ |  $$ |$$ |      $$ |      $$    |   $$ |  $$ |
   $$ |   $$$$$$$  |$$ |  $$ |$$ |      $$ |      $$$$$/    $$ |  $$ |
   $$ |   $$ |  $$ |$$ \__$$ |$$ |_____ $$ |_____ $$ |_____ $$ |__$$ |
   $$ |   $$ |  $$ |$$    $$/ $$       |$$       |$$       |$$    $$/ 
   $$/    $$/   $$/  $$$$$$/  $$$$$$$$/ $$$$$$$$/ $$$$$$$$/ $$$$$$$/

**Comment by MaxTCodes** [comment]
> I DID NOT ASK MAX.

This*

**Comment by CallumlousTV** [comment]
> > I DID NOT ASK MAX.
> 
> / |/ \ / \ / | / | / |/ \
> $$$$$$$$/ $$$$$$$ |/$$$$$$ |$$ | $$ | $$$$$$$$/ $$$$$$$ |
> $$ | $$ |**$$ |$$ | $$ |$$ | $$ | $$ |** $$ | $$ |
> $$ | $$ $$< $$ | $$ |$$ | $$ | $$ | $$ | $$ |
> $$ | $$$$$$$ |$$ | $$ |$$ | $$ | $$$$$/ $$ | $$ |
> $$ | $$ | $$ |$$ __$$ |$$ |_____ $$ |_____ $$ |_____ $$ |__$$ |
> $$ | $$ | $$ |$$ $$/ $$ |$$ |$$ |$$ $$/
> $$/ $$/ $$/ $$$$$$/ $$$$$$$$/ $$$$$$$$/ $$$$$$$$/ $$$$$$$/
```

 ________  _______    ______   __        __        ________  _______  
/        |/       \  /      \ /  |      /  |      /        |/       \ 
$$$$$$$$/ $$$$$$$  |/$$$$$$  |$$ |      $$ |      $$$$$$$$/ $$$$$$$  |
   $$ |   $$ |__$$ |$$ |  $$ |$$ |      $$ |      $$ |__    $$ |  $$ |
   $$ |   $$    $$< $$ |  $$ |$$ |      $$ |      $$    |   $$ |  $$ |
   $$ |   $$$$$$$  |$$ |  $$ |$$ |      $$ |      $$$$$/    $$ |  $$ |
   $$ |   $$ |  $$ |$$ \__$$ |$$ |_____ $$ |_____ $$ |_____ $$ |__$$ |
   $$ |   $$ |  $$ |$$    $$/ $$       |$$       |$$       |$$    $$/ 
   $$/    $$/   $$/  $$$$$$/  $$$$$$$$/ $$$$$$$$/ $$$$$$$$/ $$$$$$$/  
                                                                      
                                                                      
                                                                      
```

**Comment by CallumlousTV** [comment]
``                                                                                                                                                                                                     
                                                                                                                                                                                                     
        GGGGGGGGGGGGG        GGGGGGGGGGGGG                      IIIIIIIIIINNNNNNNN        NNNNNNNN             CCCCCCCCCCCCCHHHHHHHHH     HHHHHHHHH               AAA         TTTTTTTTTTTTTTTTTTTTTTT
     GGG::::::::::::G     GGG::::::::::::G                      I::::::::IN:::::::N       N::::::N          CCC::::::::::::CH:::::::H     H:::::::H              A:::A        T:::::::::::::::::::::T
   GG:::::::::::::::G   GG:::::::::::::::G                      I::::::::IN::::::::N      N::::::N        CC:::::::::::::::CH:::::::H     H:::::::H             A:::::A       T:::::::::::::::::::::T
  G:::::GGGGGGGG::::G  G:::::GGGGGGGG::::G                      II::::::IIN:::::::::N     N::::::N       C:::::CCCCCCCC::::CHH::::::H     H::::::HH            A:::::::A      T:::::TT:::::::TT:::::T
 G:::::G       GGGGGG G:::::G       GGGGGG    ssssssssss          I::::I  N::::::::::N    N::::::N      C:::::C       CCCCCC  H:::::H     H:::::H             A:::::::::A     TTTTTT  T:::::T  TTTTTT
G:::::G              G:::::G                ss::::::::::s         I::::I  N:::::::::::N   N::::::N     C:::::C                H:::::H     H:::::H            A:::::A:::::A            T:::::T        
G:::::G              G:::::G              ss:::::::::::::s        I::::I  N:::::::N::::N  N::::::N     C:::::C                H::::::HHHHH::::::H           A:::::A A:::::A           T:::::T        
G:::::G    GGGGGGGGGGG:::::G    GGGGGGGGGGs::::::ssss:::::s       I::::I  N::::::N N::::N N::::::N     C:::::C                H:::::::::::::::::H          A:::::A   A:::::A          T:::::T        
G:::::G    G::::::::GG:::::G    G::::::::G s:::::s  ssssss        I::::I  N::::::N  N::::N:::::::N     C:::::C                H:::::::::::::::::H         A:::::A     A:::::A         T:::::T        
G:::::G    GGGGG::::GG:::::G    GGGGG::::G   s::::::s             I::::I  N::::::N   N:::::::::::N     C:::::C                H::::::HHHHH::::::H        A:::::AAAAAAAAA:::::A        T:::::T        
G:::::G        G::::GG:::::G        G::::G      s::::::s          I::::I  N::::::N    N::::::::::N     C:::::C                H:::::H     H:::::H       A:::::::::::::::::::::A       T:::::T        
 G:::::G       G::::G G:::::G       G::::Gssssss   s:::::s        I::::I  N::::::N     N:::::::::N      C:::::C       CCCCCC  H:::::H     H:::::H      A:::::AAAAAAAAAAAAA:::::A      T:::::T        
  G:::::GGGGGGGG::::G  G:::::GGGGGGGG::::Gs:::::ssss::::::s     II::::::IIN::::::N      N::::::::N       C:::::CCCCCCCC::::CHH::::::H     H::::::HH   A:::::A             A:::::A   TT:::::::TT      
   GG:::::::::::::::G   GG:::::::::::::::Gs::::::::::::::s      I::::::::IN::::::N       N:::::::N        CC:::::::::::::::CH:::::::H     H:::::::H  A:::::A               A:::::A  T:::::::::T      
     GGG::::::GGG:::G     GGG::::::GGG:::G s:::::::::::ss       I::::::::IN::::::N        N::::::N          CCC::::::::::::CH:::::::H     H:::::::H A:::::A                 A:::::A T:::::::::T      
        GGGGGG   GGGG        GGGGGG   GGGG  sssssssssss         IIIIIIIIIINNNNNNNN         NNNNNNN             CCCCCCCCCCCCCHHHHHHHHH     HHHHHHHHHAAAAAAA                   AAAAAAATTTTTTTTTTT

**Comment by CallumlousTV** [comment]
```
                                                                                                                                                                                                     
                                                                                                                                                                                                     
        GGGGGGGGGGGGG        GGGGGGGGGGGGG                      IIIIIIIIIINNNNNNNN        NNNNNNNN             CCCCCCCCCCCCCHHHHHHHHH     HHHHHHHHH               AAA         TTTTTTTTTTTTTTTTTTTTTTT
     GGG::::::::::::G     GGG::::::::::::G                      I::::::::IN:::::::N       N::::::N          CCC::::::::::::CH:::::::H     H:::::::H              A:::A        T:::::::::::::::::::::T
   GG:::::::::::::::G   GG:::::::::::::::G                      I::::::::IN::::::::N      N::::::N        CC:::::::::::::::CH:::::::H     H:::::::H             A:::::A       T:::::::::::::::::::::T
  G:::::GGGGGGGG::::G  G:::::GGGGGGGG::::G                      II::::::IIN:::::::::N     N::::::N       C:::::CCCCCCCC::::CHH::::::H     H::::::HH            A:::::::A      T:::::TT:::::::TT:::::T
 G:::::G       GGGGGG G:::::G       GGGGGG    ssssssssss          I::::I  N::::::::::N    N::::::N      C:::::C       CCCCCC  H:::::H     H:::::H             A:::::::::A     TTTTTT  T:::::T  TTTTTT
G:::::G              G:::::G                ss::::::::::s         I::::I  N:::::::::::N   N::::::N     C:::::C                H:::::H     H:::::H            A:::::A:::::A            T:::::T        
G:::::G              G:::::G              ss:::::::::::::s        I::::I  N:::::::N::::N  N::::::N     C:::::C                H::::::HHHHH::::::H           A:::::A A:::::A           T:::::T        
G:::::G    GGGGGGGGGGG:::::G    GGGGGGGGGGs::::::ssss:::::s       I::::I  N::::::N N::::N N::::::N     C:::::C                H:::::::::::::::::H          A:::::A   A:::::A          T:::::T        
G:::::G    G::::::::GG:::::G    G::::::::G s:::::s  ssssss        I::::I  N::::::N  N::::N:::::::N     C:::::C                H:::::::::::::::::H         A:::::A     A:::::A         T:::::T        
G:::::G    GGGGG::::GG:::::G    GGGGG::::G   s::::::s             I::::I  N::::::N   N:::::::::::N     C:::::C                H::::::HHHHH::::::H        A:::::AAAAAAAAA:::::A        T:::::T        
G:::::G        G::::GG:::::G        G::::G      s::::::s          I::::I  N::::::N    N::::::::::N     C:::::C                H:::::H     H:::::H       A:::::::::::::::::::::A       T:::::T        
 G:::::G       G::::G G:::::G       G::::Gssssss   s:::::s        I::::I  N::::::N     N:::::::::N      C:::::C       CCCCCC  H:::::H     H:::::H      A:::::AAAAAAAAAAAAA:::::A      T:::::T        
  G:::::GGGGGGGG::::G  G:::::GGGGGGGG::::Gs:::::ssss::::::s     II::::::IIN::::::N      N::::::::N       C:::::CCCCCCCC::::CHH::::::H     H::::::HH   A:::::A             A:::::A   TT:::::::TT      
   GG:::::::::::::::G   GG:::::::::::::::Gs::::::::::::::s      I::::::::IN::::::N       N:::::::N        CC:::::::::::::::CH:::::::H     H:::::::H  A:::::A               A:::::A  T:::::::::T      
     GGG::::::GGG:::G     GGG::::::GGG:::G s:::::::::::ss       I::::::::IN::::::N        N::::::N          CCC::::::::::::CH:::::::H     H:::::::H A:::::A                 A:::::A T:::::::::T      
        GGGGGG   GGGG        GGGGGG   GGGG  sssssssssss         IIIIIIIIIINNNNNNNN         NNNNNNN             CCCCCCCCCCCCCHHHHHHHHH     HHHHHHHHHAAAAAAA                   AAAAAAATTTTTTTTTTT      
                                                                                                                                                                                                     
                                                                                                                                                                                                     
                                                                                                                                                                                                     
                                                                                                                                                                                                     
                                                                                                                                                                                                     
                                                                                                                                                                                                     
                                                                                                                                                                                                     
```

**Pull request merged**